osu.Game/Screens/Utility/LatencyCertifierScreen.cs

```diff
@@ -0,0 +1,436 @@
+
+            statusText.AddParagraph($"Input: {host.InputThread.Clock.FramesPerSecond} hz "
+                                    + $"Update: {host.UpdateThread.Clock.FramesPerSecond} hz "
+                                    + $"Draw: {host.DrawThread.Clock.FramesPerSecond} hz"
+                , cp => cp.Font = OsuFont.Default.With(size: 15, weight: FontWeight.SemiBold));
+
+            int certificationRemaining = !isPass ? rounds_to_complete_certified : rounds_to_complete_certified - correctCount;
+
+            if (isPass && certificationRemaining <= 0)
+            {
+                Drawable background;
+                Drawable certifiedText;
+
+                resultsArea.AddRange(new[]
+                {
+                    background = new Box
+                    {
+                        Colour = overlayColourProvider.Background4,
+                        RelativeSizeAxes = Axes.Both,
+                    },
+                    (certifiedText = new OsuSpriteText
+                    {
+                        Alpha = 0,
+                        Font = OsuFont.TorusAlternate.With(size: 80, weight: FontWeight.Bold),
+                        Text = "Certified!",
+                        Blending = BlendingParameters.Additive,
+                    }).WithEffect(new GlowEffect
+                    {
+                        Colour = overlayColourProvider.Colour1,
+                        PadExtent = true
+                    }).With(e =>
+                    {
+                        e.Anchor = Anchor.Centre;
+                        e.Origin = Anchor.Centre;
+                    }),
+                    new OsuSpriteText
+                    {
+                        Text = $"You should use a frame limiter with update rate of {mapDifficultyToTargetFrameRate(difficultyLevel + 1)} hz (or fps) for best results!",
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Font = OsuFont.Torus.With(size: 24, weight: FontWeight.SemiBold),
+                        Y = 80,
+                    }
+                });
+
+                background.FadeInFromZero(1000, Easing.OutQuint);
+
+                certifiedText.FadeInFromZero(500, Easing.InQuint);
+
+                certifiedText
+                    .ScaleTo(10)
+                    .ScaleTo(1, 600, Easing.InQuad)
+                    .Then()
+                    .ScaleTo(1.05f, 10000, Easing.OutQuint);
+                return;
+            }
+
+            string cannotIncreaseReason = string.Empty;
+
+            if (!isPass)
+                cannotIncreaseReason = "You didn't get a perfect score.";
+            else if (mapDifficultyToTargetFrameRate(difficultyLevel + 1) > target_host_update_frames)
+                cannotIncreaseReason = "You've reached the limits of this comparison mode.";
```

Comment:
I had it reading like this because it's more the limitations if the system you're running on, but I think your version probably reads better to a user.